extensions/amp-story/1.0/media-pool.js

```diff
@@ -194,19 +192,6 @@ export class MediaPool {
      */
     this.blessed_ = false;
 
-    /**
-     * The default source to use for pool-created audio sources,
-     * @private @const {!Object<!MediaType, string>}
-     */
-    this.defaultSources_ = {
-      [MediaType.AUDIO]: isExperimentOn(win, 'disable-amp-story-default-media')
```

Comment:
do we have to delete the experiment from the config or something?

Reply:
I believe experiments usage and declarations have to be removed in separate PRs

Reply:
Let's remove the experiment quickly though before it's forgotten about.

Reply:
https://github.com/ampproject/amphtml/pull/31111